### ISSUE TITLE
cellEdit do not work on firefox

### ISSUE THREAD

**Issue by ChampionRoy** [AllenFang/react-bootstrap-table#1682]
When I using Firefox browser, cell which is editable can not work when clicked,
but it works well on Chrome.
Is there anyone with the same problem?

**Comment by AllenFang** [comment]
what's the version of your firefox?

**Comment by ChampionRoy** [comment]
@AllenFang  firefox 57.0

**Comment by AllenFang** [comment]
it's wired,,,  is it work on this page > https://allenfang.github.io/react-bootstrap-table/example.html#celledit ?

**Comment by ChampionRoy** [comment]
Yeah, I tried this example and everything works well. But in my case it doesn't. I will dig out what is the difference between two cases.

**Comment by AllenFang** [comment]
@ChampionRoy feel free to reopen issue if the problem still there, thanks

**Comment by ChampionRoy** [comment]
@AllenFang  Hi, when debug both code, I found something different but I'm not sure. 
The two picture below, first is my code and second is your example code.
![default](https://user-images.githubusercontent.com/20150620/33434927-c142f8a0-d61b-11e7-9626-bb1cdf01101b.png)
![default](https://user-images.githubusercontent.com/20150620/33434937-cc0a9f5e-d61b-11e7-8354-a8a6d2fb6b2a.png)
Why my code do not bound handleEditCell() function? I'm confusing about that.

**Comment by AllenFang** [comment]
@ChampionRoy what is the `props` in your screenshot and when you print it?!

**Comment by ChampionRoy** [comment]
@AllenFang  Hi! Finally, I found that if I set blurToSave to true and set editable to a input of number type, it cannot work on firefox.  Another case is, If set blurToSave to false and set number type, it works well, and if set blurToSave to true and set another input type(not number), it works well ,too.
You can try it.

**Comment by AllenFang** [comment]
@ChampionRoy that's wired, I'll try it if I have time, thank you!!
